### DISCUSSION TITLE
Including additional data to increase the chances of purchase approval

### DISCUSSION THREAD

**Discussion by andoraul** [mercadopago/sdk-react#31] [General]
How can I include additional information such as item information and additional buyer information in my requests to increase the chances of purchase approval? What are the best practices for including this information in requests?

**Comment by francineemilia** [comment]
Hi @andoraul,

If I understand correctly your doubt you need more help with your backend - that sends the information to us - than with your frontend (which is the scope of this SDK).

In our documentation we have [some information](https://www.mercadopago.com.br/developers/en/docs/checkout-bricks/how-tos/improve-payment-approval) on how to improve your payment approval, and we encourage you to observe the respective documentation for your backend integration too.

If none of these docs help you, we suggest that you open a [support ticket](https://www.mercadopago.com/developers/en/support/center/tickets) with your doubt.

Regards,

**Comment by andoraul** [comment]
Hi @francineemilia, thanks for the response.

My doubt is for the additional frontend data, like address, and phone number. Following the instructions in the documentation [here](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/card-payment-brick/payment-submission), I've found that if I pass these additional information, the chances for a purchase approval increases. So what I am trying to do, and that's my doubt, is if I can add some data after OnSubmit was called like this sample:

```
onSubmit={async (param) => {
	const additional = {
		additional_info: {
			payer: [
				{
					phone: {
						area_code: area_code,
						number: phone_number
					}
				}
			]
		}
	}
	
	const orderData = { ...param, ...additional};
	
		return new Promise((resolve, reject) => {
		fetch(backend_url, {
			method: "POST",
			headers: {
				"Content-Type": "application/json",
			},
			body: JSON.stringify(orderData)
		})
		.then((response) => response.json())
			//check the reponse status and do something
		.catch((error) => {			
			//error
		})
	});
}}
```

**Reply by francineemilia** [reply]
Hi @andoraul 

You can do as the example you sent an add as much data you want and/or have to improve your payment approval. What is send by default in cardPayment Brick is the minimum data needed to proceed with the payment.

I reinforce that you have to pay attention to this change cause you'll have to consider the additional data and prepare your backend to deal with it.

Regards,

**Comment by andoraul** [comment]
Perfect. Thank you very much @francineemilia!
